templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py

```diff
@@ -1,44 +1,23 @@
-import azure.cosmos.cosmos_client as cosmos_client
+from azure.cosmos import CosmosClient, errors, http_constants, PartitionKey
 import azure.cosmos.errors as errors
```

Comment:
note: check if it´s needed to import errors and http_constants in both ways/lines

Reply:
Read in [here](http://www.wellho.net/mouth/418_Difference-between-import-and-from-in-Python.html) and I think we can actually get rid of those extra lines

Reply:
This will be resolved in #1618